install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+read -p "Вы уверены, что хотите установить скрипт gnu-koryavov в систему? (Д/н): " ans
```

Comment:
We'll leave it as it it for now, but later we have to consider removing user interface at all. It is very unflexible in terms of quick operation on commands, command line options must be introduced instead.

Reply:
I see